pymie/tests/test_mie.py

```diff
@@ -407,8 +407,9 @@ def test_cross_section_complex_medium():
     n_particle = Quantity(1.5+0.01j,'') 
     n_matrix = Quantity(1.0,'') 
     radius = Quantity(150,'nm')
-    theta = Quantity(np.linspace(0, np.pi, 1000), 'rad')
+    theta = Quantity(np.linspace(0.0001, np.pi, 1000), 'rad')
```

Comment:
this value shouldn't be hard-coded

Reply:
Removed the hard coded minimum, it wasn't necessary actually.